examples/evm/deposit-address/README.md

```diff
@@ -2,6 +2,17 @@
 
 This test demonstrates how to deposit tokens from one chain to another using a deposit address.
 
+
+### Context
+
+This example demonstrates how to deposit 10e6 tokens from Avalanche chain to Fantom chain using a deposit address. The initial balance of the Avalanche chain and Fantom chain is `100000000`. After the execution, the balance of Avalanche chain is `90000000` and the balance of Fantom chain is `109000000`. The deposit address is `0xb54eA64537F3307907E06d3B93ccd4A3E711623f`.
+
+A deposit address is a unique temporary address that is generated to facilitate transfers between chains. Deposit Addresses are an additional mechanism to sendToken() for sending tokens between chains. In the case of sendToken(), it is directly interacting with the gateway contract, whereas deposit addresses allow for interoperable token transfers via addresses that may be generated off chain. The goal is to allow for a similar user experience to centralized exchanges in sending funds between chains. 
```

Comment:
Add a link back to the real docs. https://docs.axelar.dev/dev/axelarjs-sdk/token-transfer-dep-addr

Reply:
Agreed good catch!